AsyncDisplayKit/ASControlNode.mm.orig

```diff
@@ -0,0 +1,683 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "ASControlNode.h"
+#import "ASControlNode+Subclasses.h"
+#import "ASThread.h"
+#import "ASDisplayNodeExtras.h"
+#import "ASImageNode.h"
+
+// UIControl allows dragging some distance outside of the control itself during
+// tracking. This value depends on the device idiom (25 or 70 points), so
+// so replicate that effect with the same values here for our own controls.
+#define kASControlNodeExpandedInset (([[UIDevice currentDevice] userInterfaceIdiom] == UIUserInterfaceIdiomPad) ? -25.0f : -70.0f)
```

Comment:
Accidentally swept in the .orig file.  Thanks for the quick resolution of the merge conflicts!  Great to see green tests.  I'm willing to land this before all the feedback is addressed if you're able to do a followup within a few days of landing.  No rush though, we can wait in the PR state if you are busy for a couple weeks.